CppCoreGuidelines.md

```diff
@@ -1042,13 +1042,14 @@ You can use the simplest "singleton" (so simple that it is often not considered
 
     X& myX()
     {
-        static X my_x {3};
-        return my_x;
+        static X* my_x = new X{3};
+        return *my_x;
     }
 
-This is one of the most effective solutions to problems related to initialization order.
+This is one of the most effective solutions to problems related to initialization and destruction order.
 In a multi-threaded environment the initialization of the static object does not introduce a race condition
-(unless you carelessly access a shared object from within its constructor).
+(unless you carelessly access a shared object from within its constructor). At destruction, the object is
```

Comment:
"At destruction, the object is not destructed" sounds strange. Maybe "At program exit, the object is not destroyed"?